### PR TITLE
Values between 0 and 100

### PR DESCRIPTION
**Pull request by Huauauaa**
image

### CODE REVIEW

**Comment by sideshowbarker** [comment]
Have you checked what the current spec actually says? For changing this in MDN, just citing a browser error message is not sufficient ー because as the existing test in MDN says, the spec says browsers are allowed to set higher or lower values than what the spec gives. Or at least it would seem that's what the spec said at the time this text was added to MDN. So need to check the current spec to see if that's what it still says, or not.

**Comment by Huauauaa** [comment]
> Have you checked what the current spec actually says? For changing this in MDN, just citing a browser error message is not sufficient ー because as the existing test in MDN says, the spec says browsers are allowed to set higher or lower values than what the spec gives. Or at least it would seem that's what the spec said at the time this text was added to MDN. So need to check the current spec to see if that's what it still says, or not.



> Have you checked what the current spec actually says? For changing this in MDN, just citing a browser error message is not sufficient ー because as the existing test in MDN says, the spec says browsers are allowed to set higher or lower values than what the spec gives. Or at least it would seem that's what the spec said at the time this text was added to MDN. So need to check the current spec to see if that's what it still says, or not.

[image](https://tc39.es/ecma262/#sec-number.prototype.toexponential)

File: files/en-us/web/javascript/reference/global_objects/number/toexponential/index.html [hunk 1]
```diff
@@ -34,7 +34,7 @@ <h3 id="Exceptions">Exceptions</h3>
 
 <dl>
  <dt>{{jsxref("RangeError")}}</dt>
- <dd>If <code><var>fractionDigits</var></code> is too small or too large. Values between <code>0</code> and <code>20</code>, inclusive, will not cause a {{jsxref("RangeError")}}. Implementations are allowed to support larger and smaller values as well.</dd>
+ <dd>If <code><var>fractionDigits</var></code> is too small or too large. Values between <code>0</code> and <code>100</code>, inclusive, will not cause a {{jsxref("RangeError")}}. Implementations are allowed to support larger and smaller values as well.</dd>
```
Comment (sideshowbarker) [inline files/en-us/web/javascript/reference/global_objects/number/toexponential/index.html:37]: Please also remove the entire sentence *Implementations are allowed to support larger and smaller values as well.*

**Review by sideshowbarker — CHANGES_REQUESTED** [review]

**Comment by sideshowbarker** [comment]
Thanks for the spec citation. So that makes it clear not only that `100` is the required upper limit of the range, but also that *Implementations are allowed to support larger and smaller values as well* is not true — because the spec doesn’t actually say that.

File: files/en-us/web/javascript/reference/global_objects/number/toexponential/index.html (hunk 1 above)
Comment (sideshowbarker) [reply files/en-us/web/javascript/reference/global_objects/number/toexponential/index.html:37]: Why did you mark this resolved?

Comment (Huauauaa) [reply files/en-us/web/javascript/reference/global_objects/number/toexponential/index.html:37]: wrong operation

Comment (Huauauaa) [reply files/en-us/web/javascript/reference/global_objects/number/toexponential/index.html:37]: and I make a [pull request](https://github.com/mdn/content/pull/874) again

**Comment by sideshowbarker** [comment]
Superseded by #874

**Pull request closed without merging**